src/containers/trends/TrendsSVGContainer.js

```diff
@@ -48,9 +48,8 @@ import { THREE_HRS } from '../../utils/datetime';
 import Background from '../../components/trends/common/Background';
 import CBGSlicesContainer from './CBGSlicesContainer';
 import SMBGsByDateContainer from './SMBGsByDateContainer';
-import SMBGRangeAvgAnimationContainer from './SMBGRangeAvgAnimationContainer';
-import SMBGAvg from '../../components/trends/smbg/SMBGAvg';
```

Comment:
Here we drop the old SMBGAvg component, but it looks like we don't pick up a new one and (as far as I can tell) we're not displaying one.